### ISSUE TITLE
Update site components and markup to use USWDS 2.0

### ISSUE THREAD

**Issue by maya** [uswds/uswds-site#646]
- [x] hero
- [x] footer
- [x] grid
- [x] sidenav
- [x] mobile navigation
  - mobile menu button on the left instead of right side 
- [x] banner hide/show text disappears

**Comment by maya** [comment]
@thisisdano this CSS is causing the govt banner button text to disappear when it's opened:
```
.usa-accordion-content[aria-hidden=true], .usa-banner-button-text--open, [aria-expanded=true]>.usa-banner-button-text {
    display: none;
}
```
Any idea why?

**Comment by thisisdano** [comment]
Huh, so this is just happening on the site, and not anywhere in Fractal? — I'll look at it soon

**Comment by maya** [comment]
@thisisdano actually it's coming from the styleguide css, i think i can figure it out.

**Comment by maya** [comment]
@thisisdano this is resolved.

**Comment by maya** [comment]
Closed in #650.
